roboteam_ai/src/coach/defence/DefencePositionCoach.cpp

```diff
@@ -137,6 +137,7 @@ std::vector<DefencePositionCoach::DefenderBot> DefencePositionCoach::decidePosit
     }
     return defenders;
 }
+*/
 world::WorldData DefencePositionCoach::removeBotFromWorld(world::WorldData world, int id, bool ourTeam) {
```

Comment:
Ah I see there is an error in this function indeed (atleast I think so). I'm not really sure how you would put the team in the lambda since we are editing the world itself, but I think I will rewrite this function regardless. I cannot make it pass by const reference since I'm editing the world here (so it's not const...). It's literally called removeBotFromWorld.

Reply:
This function is two lines.